src/sentry/issues/auto_source_code_config/code_mapping.py

```diff
@@ -403,6 +405,17 @@ def convert_stacktrace_frame_path_to_source_path(
         try_munge_frame_path(frame=frame, platform=platform, sdk_name=sdk_name) or frame.filename
     )
 
+    has_new_logic = (
+        features.has("organizations:java-frame-munging-new-logic", organization, actor=None)
+        if organization
+        else False
+    )
+    if has_new_logic and platform == "java" and frame.module and frame.abs_path:
+        try:
+            _, stacktrace_path = get_path_from_module(frame.module, frame.abs_path)
```

Comment:
`get_path_from_module` is the new logic we're using and closer to what symbolication does:
https://github.com/getsentry/sentry/blob/c86f6665bc895b371c35afcff5d2ad6794eb0d0d/src/sentry/issues/auto_source_code_config/code_mapping.py#L578-L614